### PR TITLE
Decompile and split unk_02009EAC.s

### PR DESCRIPTION
**Pull request by remicalixte**
I had to split the file in two, `UNK_021C4674` and `UNK_021C4694` were not accessed correctly otherwise. The first half deals with sprite stuff and the second with brightness so it looked like a bad split anyway

### CODE REVIEW

File: arm9/src/unk_0200A104.c [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        FUN_0200A104(&UNK_021C4694);
+    }
+
+    if (UNK_021C4674.unk1c)
+    {
+        FUN_0200A104(&UNK_021C4674);
+    }
+}
+
+THUMB_FUNC u32 FUN_0200A344(u32 param0)
```
Comment (red031000) [inline arm9/src/unk_0200A104.c:153]: BOOL?

File: include/unk_0200A104.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#ifndef POKEDIAMOND_UNK_0200A104_H
+#define POKEDIAMOND_UNK_0200A104_H
+
+#include "global.h"
+#include "nitro/FX_types.h"
+
+
+#define reg_addr_G2_BLDCNT              (u16 *)0x4000050
```
Comment (red031000) [inline include/unk_0200A104.h:8]: this should be in registers.h

**Review by red031000 — CHANGES_REQUESTED** [review]
some documentation would be nice, but idk what's possible atm

**Comment by remicalixte** [comment]
Turns out the brightness stuff was easy to figure out

**Review by red031000 — APPROVED** [review]

**Pull request merged**